src/main/resources/reference.conf

```diff
@@ -2,90 +2,88 @@
+    # setting unreachable hosts and/or not open ports wont be warned by the Kamon, your data wont go anywhere.
+    hostname = "127.0.0.1"
+    port = 8125
+
+    # Max packet size for UDP metrics data sent to Datadog.
+    max-packet-size = 1024 bytes
+    measurement-formatter = "default"
+    packetbuffer = "default"
+  }
 
-      # Max packet size for UDP metrics data sent to Datadog.
-      max-packet-size = 1024 bytes
+  #
+  # Settings relevant to the DatadogSpanReporter
+  #
+  trace {
 
-      measurement-formatter = "default"
+    # Default to agent URL (https://docs.datadoghq.com/api/?lang=python#tracing)
+    api-url = "http://localhost:8126/v0.4/traces"
 
-      packetbuffer = "default"
+    # FQCN of the "kamon.datadog.KamonDataDogTranslator" implementation that will convert Kamon Spans into Datadog
+    # Spans, or "defult" to use the built-in translator.
+    translator = "default"
 
-    }
-
-    #
-    # Settings relevant to the DatadogAPIReporter
+    # HTTP client timeout settings:
+    #   - connect-timeout: how long to wait for an HTTP connection to establish before failing the request.
+    #   - read-timeout: how long to wait for a read IO operation to complete before failing the request.
+    #   - write-timeout: how long to wait for a write IO operation to complete before failing the request.
     #
-    http {
-
-      api-url = "https://app.datadoghq.com/api/v1/series"
-
-
-      # Datadog API key to use to send metrics to datadog directly over HTTPS.
-      # If this is not set, metrics are sent as statsd packets over UDP to dogstatsd.
-      api-key = ""
+    connect-timeout = 5 seconds
+    read-timeout = 5 seconds
+    write-timeout = 5 seconds
+  }
 
-      using-agent = false
+  #
+  # Settings relevant to the DatadogAPIReporter
+  #
+  api {
 
-      connect-timeout = 5 seconds
-      read-timeout = 5 seconds
-      request-timeout = 5 seconds
-    }
+    # API endpoint to which metrics time series data will be posted.
+    api-url = "https://app.datadoghq.com/api/v1/series"
 
+    # Datadog API key to use to send metrics to Datadog directly over HTTPS. The API key will be combined with the
+    # API URL to get the complete endpoint use for posting time series to Datadog.
+    api-key = ""
 
+    # HTTP client timeout settings:
+    #   - connect-timeout: how long to wait for an HTTP connection to establish before failing the request.
+    #   - read-timeout: how long to wait for a read IO operation to complete before failing the request.
+    #   - write-timeout: how long to wait for a write IO operation to complete before failing the request.
     #
-    # Settings relevant to the DatadogSpanReporter
-    #
-    trace.http {
-
-      # Default to agent URL (https://docs.datadoghq.com/api/?lang=python#tracing)
-      api-url = "http://localhost:8126/v0.4/traces"
-
-      api-key = ${kamon.datadog.http.api-key}
-
-      using-agent = true
```

Comment:
Removed this setting because the API reporter can only send data to the API and the Spans reporter can only send data to the agent.